lib/identity_cache/query_api.rb

```diff
@@ -120,7 +120,14 @@ def check_association_scope(association_name)
         end
       end
 
-      def record_from_coder(coder) #:nodoc:
+      def record_from_coder(coder)
+        return unless coder
+        ActiveSupport::Notifications.instrument('identity_cache.hydration', class: coder[:class]) do
```

Comment:
good point, deserialization would make it sounds like it includes more and thus would be more inaccurate than hydration